PlasterTemplate/PlasterTemplates/GitLab/psake.ps1

```diff
@@ -19,45 +19,48 @@ Properties {
         "    `$PSRepository = 'https://www.powershellgallery.com/api/v2'"
     }
 %>
-
-    <#
-    $GitSettings = git config --list --show-origin
-    $GitName = ($GitSettings | Select-String -Pattern '^file:(.*?)\s+user\.name=(.*?)$').Matches.Groups
-    $GitEmail = ($GitSettings | Select-String -Pattern '^file:(.*?)\s+user\.email=(.*?)$').Matches.Groups
-    #>
-}
-
-<#
-TaskSetup {
-    if ($GitEmail -and $GitEmail[1].Value) {
-        git config -f $GitEmail[1].Value user.email 'pipeline@example.com'
-    }
-    else {
-        git config --global user.email 'pipeline@example.com'
-    }
-    if ($GitName -and $GitName[1].Value) {
-        git config -f $GitName[1].Value user.name 'pipeline'
-    }
-    else {
-        git config --global user.name 'pipeline@example.com'
+<%
+    if ($PLASTER_PARAM_DeployDocs -eq 'Yes') {
+        '    $GitSettings = git config --list --show-origin'
+        '    $GitName = ($GitSettings | Select-String -Pattern ''^file:(.*?)\s+user\.name=(.*?)$'').Matches.Groups'
+        '    $GitEmail = ($GitSettings | Select-String -Pattern ''^file:(.*?)\s+user\.email=(.*?)$'').Matches.Groups'
     }
+%>
 }
 
-TaskTearDown {
-    if ($GitEmail -and $GitEmail[2].Value) {
-        git config -f $GitEmail[1].Value user.email $GitEmail[2].Value
-    }
-    else {
-        git config --global --unset user.email
-    }
-    if ($GitName -and $GitName[2].Value) {
-        git config -f $GitName[1].Value user.name $GitName[2].Value
-    }
-    else {
-        git config --global --unset user.name
+<%
+    if ($PLASTER_PARAM_DeployDocs -eq 'Yes') {
+        '    TaskSetup {'
+        '        if ($GitEmail -and $GitEmail[1].Value) {'
+        '            git config -f $GitEmail[1].Value user.email ''pipeline@example.com'''
+        '        }'
+        '        else {'
+        '            git config --global user.email ''pipeline@example.com'''
+        '        }'
+        '        if ($GitName -and $GitName[1].Value) {'
+        '            git config -f $GitName[1].Value user.name ''pipeline'''
+        '        }'
+        '        else {'
+        '            git config --global user.name ''pipeline@example.com'''
+        '        }'
+        '    }'
+
+        '    TaskTearDown {'
+        '        if ($GitEmail -and $GitEmail[2].Value) {'
+        '            git config -f $GitEmail[1].Value user.email $GitEmail[2].Value'
+        '        }'
+        '        else {'
+        '            git config --global --unset user.email'
+        '        }'
+        '        if ($GitName -and $GitName[2].Value) {'
+        '            git config -f $GitName[1].Value user.name $GitName[2].Value'
+        '        }'
+        '        else {'
+        '            git config --global --unset user.name'
+        '        }'
+        '    }'
```

Comment:
To contradict my previous comment these actually need the indent removed. Sorry to mess you around.